build/gptfdisk/build.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/bash
+#
+# {{{ CDDL HEADER
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+# }}}
+
+# Copyright 2024 Oxide Computer Company
+
+. ../../lib/build.sh
+
+PROG=gptfdisk
+VER=1.0.9
+PKG=ooce/system/gptfdisk
+SUMMARY="GPT fdisk (gdisk, cgdisk, sgdisk, fixparts)"
+DESC="GPT fdisk is a disk partitioning tool loosely modeled on Linux "
+DESC+="fdisk, but used for modifying GUID Partition Table (GPT) disks."
+
+POPTVER=1.14
+
+set_arch 64
+
+init
+prep_build
+
+#########################################################################
+# Download and build a static version of popt
+
+save_buildenv
+
+CONFIGURE_OPTS=" --disable-shared --enable-static"
+
+build_dependency popt popt-$POPTVER popt popt $POPTVER
+
+restore_buildenv
+
+export POPT_CFLAGS="-I$DEPROOT/$PREFIX/include"
+export POPT_LDFLAGS="-L$DEPROOT/$PREFIX/lib/amd64"
+
+#########################################################################
+
+set_mirror "https://download.sourceforge.net/project/gptfdisk"
+set_checksum sha256 "dafead2693faeb8e8b97832b23407f6ed5b3219bc1784f482dd855774e2d50c2"
```

Comment:
This is useful during package development but I've put a copy of this source on the omnios download mirror; please can you remove these two lines now?

There are two main reasons for this:
- most software licences require that we make the source available for any binaries that we ship so to make this simpler we maintain a local archive where possible;
- Having all of the sources in one place allows for easier caching and building software offline (we don't always completely achieve this with some languages that love to pull things from the Internet, but we try).

Reply:
thanks! swapped over to omnios mirror